src/containers/DeveloperPortalProjects/containers/ProjectDetails/ProjectDetailsTopic/ProjectDetailsTopic.scss

```diff
@@ -0,0 +1,38 @@
+@import 'styles/font-mixins';
+
+.ProjectDetailsTopic {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: row;
+  position: relative;
+
+  @media (max-width: 480px) {
+    flex-direction: column;
+  }
+
+  &__icon {
+    margin-right: 32px;
+
+    @media (max-width: 480px) {
+      margin-bottom: 15px;
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+
+    &-main {
+      @extend %h3;
```

Comment:
this should be `%body1`?

<img width="1221" alt="Screen Shot 2021-08-14 at 9 27 08 AM" src="https://user-images.githubusercontent.com/32864116/129430447-7d8ac84c-5332-4335-91e4-f28dc90ad258.png">